src/main/resources/anniv5Resources/localization/eng/monsterhunterpack/Cardstrings.json

```
@@ -100,31 +100,31 @@
   },
   "${ModID}:SpireSpear": {
     "NAME": "Spire Spear",
-    "DESCRIPTION": "Deal !D! damage !M! times."
+    "DESCRIPTION": "Deal !D! damage. !M! times."
   },
   "${ModID}:SpireShield": {
     "NAME": "Spire Shield",
     "DESCRIPTION": "Gain !B! Block."
   },
   "${ModID}:CarvingKnife": {
     "NAME": "Carving Knife",
-    "DESCRIPTION": "Deal !D! damage. NL If this kills an Elite or Boss, gain its weapon or armor card and remove this from your deck permanently. NL Exhaust."
+    "DESCRIPTION": "Deal !D! damage. If this kills an Elite or Boss enemy, gain its weapon or armor card and remove this from your deck permanently. NL Exhaust."
   },
   "${ModID}:AwakenedRitualDagger": {
     "NAME": "Awakened Dagger",
-    "DESCRIPTION": "Deal !D! damage. NL If Fatal, heal to full HP. NL Exhaust."
+    "DESCRIPTION": "Deal !D! damage. If Fatal, heal to full HP. NL Exhaust."
   },
   "${ModID}:HuntingContract": {
     "NAME": "Hunting Contract",
     "DESCRIPTION": "Innate. NL Apply !M! ${ModID}:Hunted. NL Exhaust."
   },
   "${ModID}:HunterRank": {
     "NAME": "Hunter Rank",
-    "DESCRIPTION": "Gain !M! Strength. NL Increases by 1 for each Elite and Boss enemy you've defeated."
+    "DESCRIPTION": "Gain !M! Strength. Increases by 1 for each Elite and Boss enemy you've defeated."
   },
   "${ModID}:TranqBomb": {
     "NAME": "Tranquilizer Bomb",
-    "DESCRIPTION": "Deal !D! ${ModID}:Nonlethal damage. NL Enemy loses !M! Strength. NL Exhaust."
+    "DESCRIPTION": "Deal !D! ${ModID}:Nonlethal damage. NL Target enemy loses !M! Strength. NL Exhaust."
```

```suggestion
    "DESCRIPTION": "Deal !D! ${ModID}:Nonlethal damage. NL Target loses !M! Strength. NL Exhaust."
```
Base game precedent in Spot Weakness